app/experimenter/nimbus-ui/src/components/PageHome/Directory.tsx

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useEffect, useState } from "react";
+import { useQuery } from "@apollo/client";
+import { GET_ALL_EXPERIMENTS } from "../../gql/experiments";
+import { GetAllExperiments_experiments } from "../../types/GetAllExperiments";
+import { humanDate, addDaysToDate } from "../../lib/dateUtils";
+
+const Directory: React.FunctionComponent = () => {
+  const { data } = useQuery<{
+    experiments: GetAllExperiments_experiments[];
+  }>(GET_ALL_EXPERIMENTS);
+
+  if (!data) {
+    return <div />;
+  }
+  const { experiments } = data;
+  return (
+    <div>
+      <table className="table table-sm">
+        <thead>
+          <tr>
+            <th>Experiment</th>
+            <th>Type</th>
+            <th>Status</th>
+            <th>Start Date</th>
+            <th>End Date</th>
+            <th>Enrollment End Date</th>
+            <th>Results</th>
+          </tr>
+        </thead>
+        <tbody>
+          {experiments.map((experiment) => {
+            const {
+              name,
+              slug,
+              status,
+              startDate,
+              proposedDuration,
+              proposedEnrollment,
+              endDate,
+              featureConfig,
+            } = experiment;
+            console.log(proposedEnrollment);
+            return (
+              <tr key={slug}>
+                <td>
+                  <a href={`/experiments/nimbus/${slug}`}>{name}</a>
+                </td>
+                <td>
+                  {featureConfig ? featureConfig.slug : "Simple Experiment"}
+                </td>
+                <td>{status}</td>
+                <td>{startDate ? humanDate(startDate) : "-"}</td>
+                <td>
+                  {endDate
+                    ? humanDate(endDate)
+                    : startDate
+                    ? humanDate(addDaysToDate(startDate, proposedDuration))
+                    : `+${proposedDuration} days`}
+                </td>
+                <td>
+                  {startDate
+                    ? humanDate(addDaysToDate(startDate, proposedEnrollment))
+                    : `+${proposedEnrollment} days`}
+                </td>
+                <td>
+                  {["LIVE", "COMPLETE"].includes(status) ? (
+                    <a href={`/experiments/nimbus/${slug}/results`}>Results</a>
+                  ) : (
+                    ""
+                  )}
+                </td>
+              </tr>
+            );
+          })}
+        </tbody>
+      </table>{" "}
```

Comment:
Q: Why the explicit space after `</table>`?

Reply:
If you have Prettier enabled but you do not have the option to trim trailing whitespace enabled it may have thought the space was intentional.

Reply:
I think i copied this over from another file or something, could have been that